src/m365/spo/commands/spo-search.ts

```diff
@@ -188,6 +189,10 @@ class SpoSearchCommand extends SpoCommand {
           return `${args.options.startRow} is not a valid number`;
         }
 
+        if (args.options.startRow && args.options.allResults) {
```

Comment:
This check doesn't seem to work for me (should show error, but shows output instead):

```text
m365 spo search --queryText docs --startRow 1 --allResults

# Rows: 1
```

Reply:
Are you sure you didn't forget `npm link` or something?

![image](https://github.com/pnp/cli-microsoft365/assets/5267487/d29f30e5-127a-4c39-8f3a-13b14c7e8afb)

#itworksonmymachine 😅

Reply:
Also tried it in bash...

Reply:
Works just fine. I might've been holding it wrong 🤷‍♂️